src/actions/channels.ts

```diff
@@ -527,6 +532,9 @@ export function fetchMyChannelsAndMembers(teamId: string): ActionFunc {
             {
                 type: ChannelTypes.RECEIVED_MY_CHANNEL_MEMBERS,
                 data: channelMembers,
+                sync: true,
+                channels,
+                serverVersion: getServerVersion(getState()),
```

Comment:
this can also be simplified

```js
{
    type: ChannelTypes.RECEIVED_MY_CHANNEL_MEMBERS,
    data: channelMembers,
    sync: shouldFetchArchived,
    channels,
    remove: getChannelsIdForTeam(state, teamId) <- do we still need this???
    currentUserId,
    ....
}
````